continuous-integration/linux/create-sdist.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Could you rename this file to something with less shorthands? `create-source-distribution.sh` perhaps?

Reply:
Sure. Same for the others also then.